chapters/01.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Chapter 1
+---
+
+Here's chapter 1
+
+Here is some math: $$\mathcal{G} e^ix = \sin(x) + i \cos(x)$$
```

Comment:
Do single `$` work too? `$$` are used for centered math, not inline.

Reply:
Not yet but it seems possible.